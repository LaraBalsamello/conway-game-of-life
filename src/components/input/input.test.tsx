import React from 'react';
import { render } from '@testing-library/react';
import Input from './input';

describe('cell component should', () => {
  it('render', () => {
    const mockFunction = jest.fn();
    render(
      <Input onChange={() => mockFunction} text="this is a text" value={2} />,
    );
    const input = document.querySelector('[data-testid=container-input]');
    expect(input).toMatchSnapshot();
  });
});
