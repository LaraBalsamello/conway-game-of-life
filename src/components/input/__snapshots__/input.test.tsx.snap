// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cell component should render 1`] = `
<div
  class="input-container"
  data-testid="container-input"
>
  <label>
    this is a text
  </label>
  <input
    type="number"
    value="2"
  />
</div>
`;
